libc/src/__support/macros/properties/types.h

```diff
@@ -35,7 +35,7 @@
 #endif // UINT64_MAX
 
 // int128 / uint128 support
-#if defined(__SIZEOF_INT128__)
+#if defined(__SIZEOF_INT128__) && !defined(LIBC_TARGET_OS_IS_WINDOWS)
```

Comment:
is this necessary? I'm surprised that windows would define `sizeof` for `uint128` without defining the type

Reply:
Well clang-cl actually supports int128 types, but it emits calls into compiler-rt functions (such as `__udivti3` and `__umodti3`) to handle int128 arithmetics, which are not available in libc currently on Windows. So we disable this for now.